2017.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: page
+---
+
+### Ruby for Good 2017
+
+[Join us on Slack](https://rubyforgood.herokuapp.com/) before, during, and after the event for announcements and making friends.
+
+###### Thursday June 16th
+
+* Arrive before 5:00 pm — most of the organizing team is planning on arriving at 10:00 am but feel free to beat us there!
+* 1:00 pm - 26 people on [FONZ](https://nationalzoo.si.edu/JoinFonz/join.cfm) tour of Smithsonian-Mason Institute (see below) — [reserve your spot](https://docs.google.com/spreadsheets/d/1gTi3wUj8MJLG637wdYBsasBrca53P7X-FFjXWw-SF2k/edit?usp=sharing).
```

Comment:
We will need to create a new sign up or fix the old one.  If you want to just clear out the old google doc, I can clear it out to get it ready for a new signup.

Have the dates and times been confirmed for the tours?